x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/get_prepackaged_rules.ts

```diff
@@ -13,9 +13,7 @@ import { rawRules } from './prepackaged_rules';
  * that they are adding incorrect schema rules. Also this will auto-flush in all the default
  * aspects such as default interval of 5 minutes, default arrays, etc...
  */
-export const validateAllPrepackagedRules = (
-  rules: RuleAlertParamsRest[]
-): RuleAlertParamsRest[] => {
+export const validateAllPrepackagedRules = (rules: PrePackagedRules[]): PrePackagedRules[] => {
```

Comment:
nit:

The `PrePackagedRules` in types and the camelCasing for this function name are different capitalizations `validateAllPrepackagedRules`. This is missing the capitalization on the second 'P'.

Reply:
I changed it to use `Prepackaged` like all the rest everywhere